ixofoundation/ixo-webclient
# chore: style chatbot

## Scope
https://nonaredteam.atlassian.net/browse/IXO-599

## Work Done
Styled the chatbot summary section
Updated the selectors to match the designed content
Created buttons to close the chatbot

## Steps to Test
Make sure you can navigate the chatbot and summary sections and that they look as per the designs

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/84521255-04659880-acd5-11ea-8fba-872edcdbe482.png)
